src/content/en/fundamentals/security/credential-management/index.md

```diff
@@ -11,24 +11,26 @@ book_path: /web/fundamentals/_book.yaml
 
 The [Credential Management API](https://www.w3.org/TR/credential-management/)
 is a standards-based browser API that provides a programmatic interface
-between the site and the browser for seamless sign-in, across devices, and
-removes friction from your sign-in flows.
+between the site and the browser for seamless sign-in across devices.
 
-<div class="attempt-right">
+<!--Todo: replace this image with something much better.
+<div class="attepmt-right">
   <figure>
     <video src="animations/credential-management-smaller.mov" style="max-height: 400px;" autoplay muted loop controls></video>
     <figcaption>User Sign-in Flow</figcaption>
   </figure>
-</div>
+</div>-->
 
 The Credential Management API:
 
-* **Makes sign-in flow simple** - Users can automatically signed back into a 
+* **Removes friction from sign-in flows** - Users can automatically sign back into a 
   site, even if their session has expired.
-* **Allows one tap sign in with account chooser** - A native account chooser is shown
-  eliminating the sign in form.
-* **Stores credentials** - Can store either a username & password combination
-  or even federated account details.
+* **Allows one tap sign in with account chooser** - Users can choose an account in a native account chooser, for example Google account chooser, eliminating the sign-in form.
```

Comment:
What do you mean by "Google account chooser"? It's kind of confusing. I would remove this bit.